pkg/diagnostics/networkpod/util/log.go

```diff
@@ -0,0 +1,156 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/openshift/origin/pkg/diagnostics/types"
+	sdnplugin "github.com/openshift/origin/pkg/sdn/plugin"
+)
+
+type LogInterface struct {
+	Result types.DiagnosticResult
+}
+
+func (l *LogInterface) LogNode(logdir string) {
+	p := logdir
+
+	l.LogSystem(p)
+	l.LogServices(p)
+
+	l.Run([]string{"brctl", "show"}, "/bridges", p)
+	l.Run([]string{"docker", "ps", "-a"}, "/docker-ps", p)
+	l.Run([]string{"ovs-ofctl", "-O", "OpenFlow13", "dump-flows", sdnplugin.BR}, "/flows", p)
+	l.Run([]string{"ovs-ofctl", "-O", "OpenFlow13", "show", sdnplugin.BR}, "/ovs-show", p)
+	l.Run([]string{"tc", "qdisc", "show"}, "/tc-qdisc", p)
+	l.Run([]string{"tc", "class", "show"}, "/tc-class", p)
+	l.Run([]string{"tc", "filter", "show"}, "/tc-filter", p)
+	l.Run([]string{"systemctl", "cat", "docker.service"}, "/docker-unit-file", p)
+	l.Run([]string{"cat", "$(systemctl cat docker.service | grep EnvironmentFile.*openshift-sdn | awk -F=- '{print $2}')"}, "/docker-network-file", p)
```

Comment:
if we're porting away from being a bash script, then we should do it properly...